nextcord/application_command.py

```diff
@@ -304,11 +304,11 @@ async def handle_slash_argument(self, state: ConnectionState, argument: Any, int
             return state._get_message(int(argument))
         return argument
 
-    async def handle_message_argument(self, *args: List[Any]):
+    async def handle_message_argument(self, state: ConnectionState, argument: Any, interaction: Interaction):
         """For possible future use, will handle arguments specific to Message Commands (Context Menu type.)"""
         raise NotImplementedError  # TODO: Even worth doing? We pass in what we know already.
 
-    async def handle_user_argument(self, *args: List[Any]):
+    async def handle_user_argument(self, state: ConnectionState, argument: Any, interaction: Interaction):
         """For possible future use, will handle arguments specific to User Commands (Context Menu type.)"""
         raise NotImplementedError  # TODO: Even worth doing? We pass in what we know already.
```

Comment:
Can these be removed, since they are not even planned to be implemented in the API there is no reason to preemptively add them to Nextcord

Reply:
I'm planning on moving some of the message/user specific argument stuff to those, but the fixes present in this PR are more important than spending time doing that right now.